foundry/config.d.ts

```diff
@@ -1404,10 +1404,21 @@ declare const CONFIG: {
    * Available Weather Effects implementations
    */
   weatherEffects: {
-    leaves: any; // TODO: ConstructorOf<AutumnLeavesWeatherEffect>
-    rain: any; // TODO: ConstructorOf<RainWeatherEffect>
-    snow: any; // TODO: ConstructorOf<SnowWeatherEffect>
-  } & Partial<Record<string, any>>; // TODO: ConstructorOf<SpecialEffect>
+    /**
+     * @defaultValue `AutumnLeavesWeatherEffect`
+     */
+    leaves: ConstructorOf<SpecialEffect>;
+
+    /**
+     * @defaultValue `RainWeatherEffect`
+     */
+    rain: ConstructorOf<SpecialEffect>;
+
+    /**
+     * @defaultValue `SnowEatherEffect`
+     */
+    snow: ConstructorOf<SpecialEffect>;
```

Comment:
Shouldn't those be more specific?

Reply:
People could override these effects with effects that they created themselves that don't actually inherit from the more specific effect classes. So I don't think it should be more specific. You don't get any benefit from that anyways.